front/src/applications/operationalStudies/components/SimulationResults/SpeedSpaceChart/SpeedSpaceChart.jsx

```diff
@@ -101,8 +101,7 @@ export default function SpeedSpaceChart(props) {
     let debounceTimeoutId;
     clearTimeout(debounceTimeoutId);
     debounceTimeoutId = setTimeout(() => {
-      const height = d3.select(`#container-${CHART_ID}`).node().clientHeight;
-      setHeightOfSpeedSpaceChart(height);
+      createChartAndTrain();
```

Comment:
I'm not sure It work the same. Not this behavior on dev (Storybook) 

https://user-images.githubusercontent.com/765626/226406192-a04a945d-718f-44fe-8a2d-5d9c9e4646ea.mp4